spec/datetime_helper_spec.rb

```diff
@@ -8,6 +8,12 @@
       it { expect(DatetimeHelper.is_zulu_time?(valid_string)).to eq true }
     end
 
+    context "given a valid Zulu Datetime string" do
+      let(:valid_string) { DateTime.now.new_offset(0).iso8601 }
```

Comment:
Mind change this to an actual string so it's obvious what the valid date-time format looks like in this case.